Automattic/sensei
# Add wp-env support

### Changes proposed in this Pull Request

This PR adds [wp-env](https://developer.wordpress.org/block-editor/reference-guides/packages/packages-env/) support. This could be useful to some people that don't have the time or knowledge to set up a dev environment but still want to play around with the plugin or contribute.

### Testing instructions

* Have Docker installed and started.
* Run `npm install`.
* Run `npm run wp-env start`.
* Open http://localhost:8888/ and make sure the plugin is installed.
